shadows/framework/src/main/java/org/robolectric/shadows/ShadowAppOpsManager.java

```diff
@@ -285,7 +289,7 @@ protected int checkOp(String op, int uid, String packageName) {
    * String, String, String)} without {@link #finishOp(String, int, String, String)} yet.
    */
   @Implementation(minSdk = R)
-  protected boolean isOpActive(String op, int uid, String packageName) {
+  public boolean isOpActive(String op, int uid, String packageName) {
```

Comment:
Hi @hoisie , does it permitted for implementation method? This is only bound to API from R, and what about adding a new method for normal usage instead of making it public if there is need to use it?

Reply:
Looks like this function works with `longRunningOp`, which gets populated pre-R as well:

https://github.com/robolectric/robolectric/blob/master/shadows/framework/src/main/java/org/robolectric/shadows/ShadowAppOpsManager.java#L253

I agree it's a little confusing. Thoughts @sooniln?

Reply:
I will merge this for now, to unblock syncing other changes, but I'll chat w/ @sooniln about this to get clarity.